bundle/manifests/amalthea.dev_amaltheasessions.yaml

```diff
@@ -5985,6 +5985,20 @@ spec:
                   - name
                   type: object
                 type: array
+              location:
```

Comment:
The changes in this file are just leftovers from other PRs that are already merged. They CRDs in the helm chart already have this update. This is for deploying directly from manifests.